pyvista/core/pointset.py

```diff
@@ -2518,8 +2518,8 @@ def dimensions(self) -> Tuple[int, int, int]:
         Examples
         --------
         >>> from pyvista import examples
-        >>> grid = examples.load_explicit_structured()  # doctest:+SKIP
-        >>> grid.dimensions  # doctest:+SKIP
+        >>> grid = examples.load_explicit_structured()  # pragma: no cover
+        >>> grid.dimensions  # pragma: no cover
```

Comment:
According to `git grep -E '>>> .+ # pragma'` this is the only such comment in doctests. What is the reason for this change? (Contrast with `git grep -E '>>> .+ # doctest'` that returns a bunch of results.)

Reply:
> What is the reason for this change?

Attempting to code while jetlagged on a mobile device.

Reply:
Interestingly enough, this still ran.